modules/swagger-codegen/src/main/resources/Java/ApiClient.mustache

```diff
@@ -61,10 +55,9 @@ public class ApiClient {
     setUserAgent("Java-Swagger");
 
     // Setup authentications (key: authentication name, value: authentication).
-    authentications = new HashMap<String, Authentication>();{{#authMethods}}{{#isBasic}}
-    authentications.put("{{name}}", new HttpBasicAuth());{{/isBasic}}{{#isApiKey}}
-    authentications.put("{{name}}", new ApiKeyAuth({{#isKeyInHeader}}"header"{{/isKeyInHeader}}{{^isKeyInHeader}}"query"{{/isKeyInHeader}}, "{{keyParamName}}"));{{/isApiKey}}{{#isOAuth}}
-    authentications.put("{{name}}", new OAuth());{{/isOAuth}}{{/authMethods}}
+    authentications = new HashMap<String, Authentication>();
```

Comment:
We were experimenting with using this branch to get up-and-running with jersey2, and thought this hard-coded change might be here because of development in trying to get the authentication pieces working with jersey2. This is a friendly reminder that this specific part of the changeset should probably be reverted, as well as the change to [`basePath`](https://github.com/swagger-api/swagger-codegen/pull/950/files#diff-3714cb3e3c7bbb1ce24de92a3fb9f01bR40)